### ISSUE TITLE
Gamepad compatibility

### ISSUE THREAD

**Issue by Snayler** [RustyMods/PortalStations#3]
Would it be possible to make it gamepad compatible? Like Joystick to chose portal, A to teleport, B to close dialog, etc?

**Comment by keckstein14** [comment]
Yes this would be awesome! I'm trying to find an "any destination" portal mod that has gamepad support but haven't been able to find one yet.
